src/Styleguide/Pages/Artist/Routes/AuctionResults/AuctionResultItem.tsx

```diff
@@ -13,32 +16,61 @@ import { Responsive } from "Styleguide/Utils/Responsive"
 import { Subscribe } from "unstated"
 import { AuctionResultsStateContainer } from "./AuctionResultsState"
 
-export interface Props {
+export interface Props extends ContextProps {
```

Comment:
Added this but as mentioned below, wound up just using the user in the global state, so maybe will undo.

Reply:
Yeah I think it's much more convenient to tap into GlobalState for user stuff.